### ISSUE TITLE
Incorrect node requirements

### ISSUE THREAD

**Issue by D4nte** [alexbosworth/balanceofsatoshis#8]
Hi,

As per the readme: `Requires an installation of Node v10.12.0+ and NPM`.

However, this is incorrect: 
```
▶ yarn add balanceofsatoshis
yarn add v1.22.0
[1/5] 🔍  Validating package.json...
[2/5] 🔍  Resolving packages...
warning balanceofsatoshis > crypto-js@3.2.0: Critical
[3/5] 🚚  Fetching packages...
error lightning@1.2.0: The engine "node" is incompatible with this module. Expected version ">=12.0.0". Got "10.19.0"
error Found incompatible module.
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
```

I believe it is because it depends on `ln-accounting@3.1.8` > `ln-service@47.11.0` > `lightning@1.2.0`.

The node compatibility was corrected in `lightning@1.2.2`.

If it is possible to upgrade the dependencies then it would resolve the issue.

**Comment by alexbosworth** [comment]
Ok I'll take a look

**Comment by alexbosworth** [comment]
Can you try it now?

**Comment by D4nte** [comment]
Yep! It is fixed (5.19.1 got installed). Thanks!
